src/components/post-card/post-card.module.scss

```diff
@@ -1,112 +1,229 @@
-	border-radius: var(--corner-radius_xl);
+.postContainer {
+	padding: var(--card_padding);
+}
 
-	overflow-wrap: break-word;
-	max-width: 100%;
+.postHeaderBase {
+	color: var(--card_title_color);
+	margin: 0;
 	text-decoration: none;
-	@include transition(background-color border-color);
+}
+
+.postHeaderBase > h2 {
+	margin: 0;
+}
 
-	padding: var(--spc-6x);
+.postDataContainer {
+	padding: var(--card_detail-container_padding-vertical) 0;
 	display: flex;
 	flex-direction: column;
-	position: relative;
-	justify-content: flex-end;
-	overflow: hidden;
-
-	cursor: pointer;
+	gap: var(--card_detail-container_gap);
+}
 
-	&:hover {
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: transparent;
-	}
+.authorList {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	display: flex;
+	flex-direction: row;
+}
 
-	&:active {
-		background-color: var(--surface_primary_emphasis-low);
-		border-color: var(--surface_primary_emphasis-high);
-	}
+.authorName {
+	margin: 0;
+	text-decoration: none;
+}
 
-	&:focus {
-		background-color: var(--background_focus);
-		border-color: var(--focus-outline_primary);
-	}
+.authorListContainer {
+	display: flex;
+	flex-direction: row;
+	gap: var(--card_detail-row_gap);
+	align-items: center;
 }
 
-.image {
-	position: absolute;
-	top: 0;
-	left: 50%;
-	transform: translateX(-50%);
+.cardIcon {
+	padding: var(--icon-size-dense-padding);
+	color: var(--card_detail-row_icon_color);
+	height: var(--card_detail-row_icon_size);
+	width: var(--card_detail-row_icon_size);
+}
 
+.cardIcon > svg {
 	height: 100%;
-	opacity: 0.5;
+	width: 100%;
+}
 
-	z-index: -1;
-	mask-image: linear-gradient(to bottom, #000F, #0000);
-	-webkit-mask-image: linear-gradient(to bottom, #000F, #0000);
-	background-color: #FFF;
+.dateAndWordCount {
+	display: flex;
+	flex-direction: row;
+	gap: var(--card_detail-row_gap);
+	align-items: center;
+}
 
-	@include darkTheme() {
-		filter: invert(1);
-		background-color: #000;
-	}
+.publishedDate {
+	margin: 0;
+	color: var(--card_detail-row_details_color);
 }
 
-.meta {
-	margin: var(--spc-4x) 0;
+.separatorDot {
+	margin: 0;
+	color: var(--card_detail-row_dot_color);
+}
 
-	&> * {
-		margin: 0;
-		line-height: 1;
-	}
+.wordCount {
+	margin: 0;
+	color: var(--card_detail-row_details_color);
+}
 
-	.date {
-		color: var(--foreground_emphasis-medium);
-	}
+.description {
+	margin: 0;
+	color: var(--card_snippet_color);
 }
 
-.header {
-	display: block;
-	text-decoration: none;
-	color: var(--foreground_emphasis-high);
+.cardList {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	display: flex;
+	flex-direction: row;
+	gap: var(--card_tags_gap);
+	margin-top: var(--card_tags_padding-top);
+}
 
-	& h2 {
-		margin: 0;
-	}
+.spacer {
+	height: 1px;
+	flex-grow: 1;
 }
 
-.excerpt {
-	margin: 0;
-	margin-bottom: var(--spc-4x);
+.extendedPostContainer,
+.regularPostContainer {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+}
+
+.extendedPostImageContainer {
+	height: 1px;
```

Comment:
This is still rendering the image on the mobile breakpoint with a height of `1px` (which is more visible in dark mode...) - I think this should have a `@include from($tabletLarge)` to make it `display: none;` on mobile.